clients-simple/src/main/resources/application.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2020 Crown Copyright
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+web:
+  client:
+    palisade-service: "palisade-service-not-configured"
+    data-service: "data-service-not-configured"
```

Comment:
@dev-958 's preference seemed to be to use the static config here

Reply:
yes please